Automattic/jetpack
# Update Midnight color scheme

Follows up https://github.com/Automattic/wp-calypso/pull/59461.

#### Changes proposed in this Pull Request:

Aligns the Midnight color scheme used by Nav Unification with the changes introduced in https://github.com/Automattic/wp-calypso/pull/59461, so the blue shades are replaced with red shades from Color Studio.

Before | After
--- | ---
<img width="312" alt="Screen Shot 2021-12-22 at 11 56 45" src="https://user-images.githubusercontent.com/1233880/147082147-132fca70-eabd-4efe-ab5b-88d99e0499fd.png"> | <img width="321" alt="Screen Shot 2021-12-22 at 11 56 24" src="https://user-images.githubusercontent.com/1233880/147082172-cdd283e1-82cc-4991-bc8c-0ded100cdc19.png">

#### Jetpack product discussion
N/A.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
- Prepare your testing environment:
  - Simple sites: Apply D72133-code to your sandbox, and sandbox a Simple site.
  - Atomic sites: Install Jetpack Beta and activate the `update/masterbar-color-scheme-midnight` branch.
- Go to https://wordpress.com/me/account.
- Switch to the Midnight color scheme.
- Go to a WP Admin page of your test site.
- Make sure the inline help icon in the bottom right corner uses red shades instead of blue shades.
